daemon-tests/tests/collaborative_settlement.rs

```diff
@@ -0,0 +1,102 @@
+use daemon::projection::CfdState;
+use daemon_tests::confirm;
+use daemon_tests::dummy_quote;
+use daemon_tests::flow::next_with;
+use daemon_tests::flow::one_cfd_with_state;
+use daemon_tests::maia::OliviaData;
+use daemon_tests::start_from_open_cfd_state;
+use daemon_tests::wait_next_state;
+use model::Position;
+use otel_tests::otel_test;
+use std::time::Duration;
+use tokio_extras::time::sleep;
+
+#[otel_test]
+async fn collaboratively_close_an_open_cfd_maker_going_short() {
+    collaboratively_close_an_open_cfd(Position::Short).await;
+}
+
+#[otel_test]
+async fn collaboratively_close_an_open_cfd_maker_going_long() {
+    collaboratively_close_an_open_cfd(Position::Long).await;
+}
+
+async fn collaboratively_close_an_open_cfd(maker_position: Position) {
```

Comment:
🙃 I think it's better in test files when all the tests are at the top, and all the helpers at the bottom.